### ISSUE TITLE
Operations missing on PawnKinds added with HAR when WikiRim is also loaded

### ISSUE THREAD

**Issue by Bbansjkl** [SamboyCoding/RimworldBetterLoading#10]
Okay, this is a weird one, and I have absolutely no idea why this is happening.

When this mod is loaded together with Humanoid Alien Races, an alien race of some kind and WikiRim, then alien races will not have the default operations, like amputating, harvesting, adding artificial bodyparts, anesthesics etc.  
This works fine when either WikiRim or BetterLoading are not being loaded. There isn't anything of importance in the logs. Here's a copy of the ingame log: [Link](https://pastebin.com/RG8QHvjX)  
The only error is the one for the empty float menu at the bottom, where I clicked on "Add bill" on one of the alien pawn's Health>Operations tab.

Reproduction:
1) Start game with the following mods:

  - brrainz.harmony
  - me.samboycoding.betterloading
  - Ludeon.RimWorld
  - Ludeon.RimWorld.Royalty
  - Mlie.WikiRim [Steam](https://steamcommunity.com/sharedfiles/filedetails/?id=2198532017) [Github](https://github.com/emipa606/WikiRim)
  - erdelf.HumanoidAlienRaces [Steam](https://steamcommunity.com/sharedfiles/filedetails/?id=839005762) [GitHub](https://github.com/RimWorld-CCL-Reborn/AlienRaces)
  - Razar1.MonstergirlRaces [Steam](https://steamcommunity.com/sharedfiles/filedetails/?id=1992614103)

Note: you can probably use any other mod adding new races, too. I tested it with MGR and [Ikquan Race](https://steamcommunity.com/sharedfiles/filedetails/?id=1663859834), to make sure it isn't just an issue with MGR.

2) Start a new game and reroll your starting colonists until you get at least one non-human pawn.  
3) Start a colony.  
4) Select a non-human pawn and check the operations menu.

Operations added by other mods seem to be fine, only vanilla ones are affected. I've tried to load the Wiki before Core, but that doesn't make a difference.

Other:  
Regarding the log: I currently use a slightly modified version of WikiRim. There's at the moment a slight issue with the MG races because their corpses don't have a leather amount set. WikiRim thus will show a warning and not build a help page for them. Since I first thought that to be the reason for this error, I fixed it in a local build by checking if leatherAmount is defined and ignoring it if not, but this error here persists. I'm going to post a PR for that to WikiRim later today.

Not sure if this issue actually has its root here, but since WikiRim doesn't alter anything not in its domain and BetterLoading is the more "invasive" mod of the two, I'm thinking that it's more likely you would be able to figure out what's going wrong here.

**Comment by SamboyCoding** [comment]
Yeah that's certainly an unusual issue. I'll try to see if the mods are doing anything weird, but I've had issues with HAR before and they're not easy to solve.
